server/views/pages/prisons/addSessionTemplate.njk

```diff
@@ -0,0 +1,254 @@
+{% extends "../../partials/layout.njk" %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{%- from "govuk/components/back-link/macro.njk" import govukBackLink -%}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+
+{% set pageHeaderTitle = 'Add session template' %}
+{% set pageTitle = applicationName + " - " + pageHeaderTitle %}
+
+{% set primaryNavItems = [
+  {
+    text: "Home",
+    href: "/"
+  },
+  {
+    text: "Prisons",
+    href: "/prisons",
+    active: true
+  }
+  ] %}
+{% set backLinkHref = "/prisons/" + prisonId + "/session-templates" %}
+
+{% set endDateTextHtml %}
+  {{ govukDateInput({
+    id: "validToDate",
+    namePrefix: "validToDate",
+    fieldset: {
+      legend: {
+        text: "Template end date",
+        classes: "govuk-fieldset__legend--m"
+      }
+    },
+    hint: {
+      text: "For example, 27 3 2023"
+    }
+  }) }}
+{% endset -%}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ prisonName }}</span>
+        {{ pageHeaderTitle }}
+      </h1>
+
+      <form action="/prisons/{{prisonId}}/session-templates/add" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+        {{ govukInput({
+          label: {
+              text: "Template name",
+              classes: "govuk-label--m"
+          },
+          classes: "govuk-!-width-three-quarters",
+          id: "templateName",
+          name: "templateName",
+          autocomplete: "off",
```

Comment:
Might be worth consistently turning off both `autocomplete` and `spellcheck` for all these text inputs. Can't see that you'd ever want it.